frontend/src/styles/global.scss

```diff
@@ -684,3 +684,23 @@ body {
         display: none;
     }
 }
+
+.line-clamp-2 {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+}
+
+.object-cover {
+    object-fit: cover;
+    object-position: center;
+}
+
+.h-120 {
```

Comment:
same with these two except we're using the numbers differently... for all other `h`(and similar) values `X` doesn't map to `Xpx`

They're numbers in rem*4 i.e. `h-8` maps to `2rem`

Here these would map to roughly `h-24` and `h-28`